src/components/admonition/admonition.tsx

```diff
@@ -0,0 +1,86 @@
+import { Component, h, Prop } from '@stencil/core';
+import { AdmonitionType } from '@limetech/lime-elements';
+
+/**
+ * @exampleComponent limel-example-admonition-simple-warning
+ * @exampleComponent limel-example-admonition-long-information
+ * @exampleComponent limel-example-admonition-composite
+ */
+@Component({
+    tag: 'limel-admonition',
+    shadow: true,
+    styleUrl: 'admonition.scss',
+})
+export class Admonition {
+    /**
+     * Title of the admonition
+     */
+    @Prop({ reflect: true })
+    public title?: string;
+
+    /**
+     * Description of the admonition
+     */
+    @Prop({ reflect: true })
+    public description?: string;
+
+    @Prop({ reflect: true })
+    public type: AdmonitionType = 'info';
+
+    public render() {
+        return (
+            <div class={{ admonition: true, [this.type]: true }}>
+                <div class="admonition-side">
+                    <limel-icon
+                        name={this.getIcon()}
+                        size="medium"
+                    ></limel-icon>
+                </div>
+                <div class="admonition-main">
+                    <h3 class="admonition-main-title">{this.getTitle()}</h3>
+                    <p class="admonition-main-description">
+                        {this.description}
+                    </p>
+                </div>
+            </div>
+        );
+    }
+
+    private getIcon(): string {
+        switch (this.type) {
+            case 'danger':
+                return 'fire_element';
+            case 'info':
+                return 'info';
+            case 'note':
+                return 'info';
+            case 'tip':
+                return 'light_on';
+            case 'warning':
+                return 'error';
```

Comment:
The `danger` icon is nice, my only small objection is that it is perhaps a bit _too_ nice. The `warning` icon looks more dangerous than the `danger` icon 😅

Reply:
We also have a legal problem of redistributing icons form icons8 lib.
So, we must do something about this. If we reuse icons from Kompendium, they're done by myself. So it's oK. If we wanna allow consumers to add their own icons, then we have to figure out a way to allow that. Maybe using _CSS part_ or like the lime admin hack we did for including customer logotypes.

Reply:
> We also have a legal problem of redistributing icons form icons8 lib.

Yeah, but we're not including the icons here, we only set the names of the icons to use. A consumer that doesn't have access to `lime-icons8` will just have to provide their own icons with these names.

Reply:
gotcha